lib/internal/process/esm_loader.js

```diff
@@ -2,13 +2,13 @@
 
 const {
   ArrayIsArray,
-  ObjectCreate,
 } = primordials;
 
 const {
   ERR_VM_DYNAMIC_IMPORT_CALLBACK_MISSING,
 } = require('internal/errors').codes;
 const { ESMLoader } = require('internal/modules/esm/loader');
+const publicESMLoader = require('internal/modules/esm/public_loader_proxy');
```

Comment:
What does “public” mean here? As in there are two `ESMLoader` instances, one for Node internals and one for user code, and this is the latter? Is it publicly accessible to user code?

Reply:
Correct: internal is used internally by node for its own code. Public handles userland code. Neither are accessible in userland.

Reply:
If it’s not accessible to application code, “public” feels like a poor choice of name. Maybe `applicationESMLoader` or `userCodeESMLoader`?